pkg/cluster/internal/create/create.go

```diff
@@ -249,8 +254,11 @@ func validateProvider(p providers.Provider) error {
 		if !info.Cgroup2 {
 			return errors.New("running kind with rootless provider requires cgroup v2, see https://kind.sigs.k8s.io/docs/user/rootless/")
 		}
-		if !info.SupportsMemoryLimit || !info.SupportsPidsLimit || !info.SupportsCPUShares {
-			return errors.New("running kind with rootless provider requires setting systemd property \"Delegate=yes\", see https://kind.sigs.k8s.io/docs/user/rootless/")
+		// Do not check systemd if running kind in sandbox workspace.
+		if !crafting.InSandboxWorkspace() {
```

Comment:
Can we set `SupportsMemoryLimit`, `SupportsPidsLimit` and `SupportsCPUShares` to `true` in crafting provider and avoid changing this?